twake/backend/node/src/services/channels/web/schemas.ts

```diff
@@ -2,34 +2,105 @@ export const createChannelSchema = {
   body: {
     type: "object",
     properties: {
-      creator: {
-        type: "string",
+      resource: {
+        type: "object",
```

Comment:
OK, going to push something to be able to share schemas so that we do not dup them